components/route/route-component.ts

```diff
@@ -241,15 +243,18 @@ export class BaseRouteComponent extends Vue {
 
 		// Set up to watch the route title change.
 		if (this.routeTitle) {
-			Meta.title = this.routeTitle;
+			Meta.setTitle(this.routeTitle, this.disableRouteTitleSuffix);
 		}
 
 		this.$watch('routeTitle', (title: string | null) => {
 			if (title) {
-				Meta.title = title;
+				Meta.setTitle(title, this.disableRouteTitleSuffix);
 			}
 		});
 
+		// TODO: should I watch disableRouteTitleSuffix?
```

Comment:
You should figure this TODO out before merging

Reply:
I left it for the PR review. The issue is we watch routeTitle to figure out when we should set the Meta title. usually only within these funcs we'll modify the disableRouteTitleSuffix variable so the question is if that watch is ever needed. I can't currently think of a case where we'd modify disableRouteTitleSuffix outside of routeTitle.